pkg/relayer/proxy/proxy_test.go

```diff
@@ -462,6 +473,50 @@ func TestRelayerProxy_Relays(t *testing.T) {
 			expectedErrCode: 0,
 			expectedErrMsg:  "",
 		},
+		{
+			desc: "Successful late relay with session grace period",
+
+			relayerProxyBehavior: []func(*testproxy.TestBehavior){
+				// blockHeight is past the first session but within its session grace period
+				testproxy.WithRelayerProxyDependenciesForBlockHeight(
+					supplierKeyName,
+					blockHeight+sessionkeeper.GetSessionGracePeriodBlockCount(),
```

Comment:
Consider moving this above to `blockWithinSessionGracePeriod := ` above, to make the two variables really easy to read side-by-side.